acceptance/config/aio/options.rb

```diff
@@ -1,9 +1,13 @@
 {
   :type => 'aio',
+  :is_puppetserver => true,
+  :puppetservice => 'puppetserver',
+  :'puppetserver-confdir' => '/etc/puppetlabs/puppetserver/conf.d',
   :pre_suite => [
     'setup/common/000-delete-puppet-when-none.rb',
     'setup/aio/pre-suite/010_Install.rb',
     'setup/aio/021_InstallAristaModule.rb',
     'setup/common/010_Setup_Broker.rb',
+    'setup/common/040_ValidateSignCert.rb'
```

Comment:
Ouch, thanks for catching that. I've added it then squashed the commit so no-one will ever know